core/include/kernel/tee_ta_manager.h

```diff
@@ -152,4 +170,12 @@ void tee_ta_dump_current(void);
 TEE_Result tee_ta_verify_param(struct tee_ta_session *sess,
 			       struct tee_ta_param *param);
 
+#if defined(CFG_TA_GPROF_SUPPORT)
+void tee_ta_gprof_sample_pc(vaddr_t pc);
+void tee_ta_update_session_utime(bool suspend);
```

Comment:
The code would be easier to follow if this function either took a descriptive enum instead of a bool or if there was two functions, `tee_ta_update_session_utime_suspend()` and `tee_ta_update_session_utime_resume()`. I'm leaning towards the latter, but it's no big deal.